content/_code-samples/send-a-memo/py/send-a-memo.py

```diff
@@ -0,0 +1,61 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import Payment, Memo
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.wallet import generate_faucet_wallet
+
+# This code sample validates and sends a transaction with a Memo attached
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/transaction-common-fields.html#memos-field
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Get account credentials from the Testnet Faucet
+print("Requesting account from the Testnet faucet...")
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Example: TEXT 321 -+=
+memo_data = input("Enter memo data to insert into a transaction: ")
+# Example: text
+memo_type = input("Enter memo type to insert into a transaction: ")
```

Comment:
This field doesn't make as much sense to allow the user to input. (I'm at least not sure what options make sense to put in there) - While it does technically allow arbitrary input, I'd recommend either setting this for the user to `Text` to keep it simple, or to add a print statement explaining what this field is used for - Ex. 
print("Memo Type is used to describe the type of data you're sharing. Common types include 'Text', '...', and '...'")

Reply:
Ye I agree, I'll set it to use a variable as the Memo